data/transactions/logic/eval.go

```diff
@@ -4038,7 +4526,7 @@ func (cx *EvalContext) PcDetails() (pc int, dis string) {
 
 func base64Decode(encoded []byte, encoding *base64.Encoding) ([]byte, error) {
 	decoded := make([]byte, encoding.DecodedLen(len(encoded)))
-	n, err := encoding.Strict().Decode(decoded, encoded)
+	n, err := encoding.Decode(decoded, encoded)
```

Comment:
this would break compatibility if it wasn't for `fidoVersion`

Reply:
I convinced myself that Strict() was the default, and unneeded, and none of our test complained.  However, I'm no longer so sure, so I will try to create tests that care about this.

Reply:
Test added.  We'll debate and decide whether we want Strict() or not before we adjust fidoVersion.